internal/hns/namespace.go

```diff
@@ -27,9 +27,10 @@ type namespaceResourceRequest struct {
 }
 
 type Namespace struct {
-	ID           string
-	IsDefault    bool                `json:",omitempty"`
-	ResourceList []NamespaceResource `json:",omitempty"`
+	CompartmentId uint32
+	ID            string
+	IsDefault     bool                `json:",omitempty"`
+	ResourceList  []NamespaceResource `json:",omitempty"`
```

Comment:
the latter two fields have json tags with them, do we need that on the compartment ID field to prevent breaking anything else relying on this or?

Reply:
I'll add a tag. Marshalling CompartmentId as 0 if it's not present yet would not be good :)

Reply:
PTAL